### DISCUSSION TITLE
Episode 182: C++ Variadic Templates, Swift and More with Doug Gregor

### DISCUSSION THREAD

**Discussion by codereport** [codereport/adsp2#77] [Episode]
Episode Link: https://adspthepodcast.com/2024/05/17/Episode-182.html

Feel free to write comments or ask questions 🙂

**Comment by jdebache** [comment]
Great episode. Two things stand out to me: discriminated unions and optional types should have language support. Once you've worked with a language where this is true, it's difficult to go back.
